dynosaur_derive/src/lib.rs

```diff
@@ -53,173 +49,91 @@ pub fn dynosaur(
     item: proc_macro::TokenStream,
 ) -> proc_macro::TokenStream {
     let _attrs = parse_macro_input!(attr as Attrs);
-    let item = parse_macro_input!(item as ItemTrait);
+    let item_trait = parse_macro_input!(item as ItemTrait);
 
-    let erased_trait = mk_erased_trait(&item);
+    let expanded_trait = expand_trait(&item_trait);
+    let erased_trait = mk_erased_trait(&expanded_trait);
+    let erased_trait_blanket_impl =
+        mk_erased_trait_blanket_impl(&expanded_trait.ident, &erased_trait);
```

Comment:
`item_trait.ident` would work too right? I think that makes more sense.

Reply:
Yes, it is basically the same trait but with async fns expanded.